broker/src/main/java/io/zeebe/broker/system/partitions/impl/steps/ExporterDirectorPartitionStep.java

```diff
@@ -31,6 +31,8 @@
 
     final ExporterDirector director = new ExporterDirector(exporterCtx, !context.shouldExport());
     context.setExporterDirector(director);
+    director.addFailureListener(context.getPartition());
```

Comment:
```suggestion
    context
                .getComponentHealthMonitor()
                .registerComponent(director.getName(), director);
```

ZeebePartition is not monitoring the components directly, instead it has healthmonitor that periodically checks all the components that are installed by ZeebePartition.